src/jsonformat.erl

```diff
@@ -52,7 +53,19 @@ format(Map = #{msg := {string, String}}, Config) ->
   Report = #{text => unicode:characters_to_binary(String)},
   format(Map#{msg := {report, Report}}, Config);
 format(Map = #{msg := {Format, Terms}}, Config) ->
-  format(Map#{msg := {string, io_lib:format(Format, Terms)}}, Config).
+  format(Map#{msg := {string, io_lib:format(Format, Terms)}}, Config);
+format(Unknown, Config) ->
+  erlang:display({Unknown, Config}).
+
+
+-spec unix_to_iso8601(integer()) -> binary().
+unix_to_iso8601(Epoch) ->
+  unix_to_iso8601(Epoch, microsecond).
+
+-spec unix_to_iso8601(integer(), erlang:time_unit()) -> binary().
```

Comment:
do we need this function here (and exported)? `unix_to_iso8601/1` should be sufficient, or?